main/test/resources/examples/foreign/project/build.sc

```diff
@@ -56,6 +57,13 @@ def checkOuterInnerPaths = T {
   )
 }
 
+def checkOtherPaths = T {
```

Comment:
Is this used? I can't find any usage in this PR

Reply:
Good catch. This slipped through in my last selective commit.